integration-test/run.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+# Test deploying a function to AWS and running it.
+
+set -e
+
+for DEPENDENCY in curl jq npm stack
+do
+    which $DEPENDENCY >/dev/null || \
+        (echo "$DEPENDENCY is required for the test." >&2; exit 1)
+done
+
+# Root directory of the repository
+DIST=$(cd $(dirname $0)/..; echo $PWD)
+
+# Directory with the integration test files
+TEST=$(cd $(dirname $0); echo $PWD)
+
+# Stackage resolver to use
+RESOLVER=$(curl -s https://www.stackage.org/download/snapshots.json | \
```

Comment:
You should document the dependencies for the integration test, e.g. `curl`, `jq`, `sls`, etc.